### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by samuelsogbesan**
Create a barebones README.md to describe the purpose and functionality of the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Portfolio (STEP Project)
+
+## Introduction
+This is the repository hosting the files for the repository I built as my first project for the STEP program!
+Once deployed, I will attach a URL at which it can viewed.
+
+## Screenshots
```
Comment (krasimirgg) [inline README.md]: nit: don't leave empty sections. you can always add them up with contexts later, but it's very easy to forget about them and never ever touch them in the future.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Installation
+
+## Contributing
+Contributions and fixes are welcome. Before opening a PR with changes, please open an issue first to discuss anything you'd want to change!
+
+## License
```
Comment (krasimirgg) [inline README.md]: I believe the original repository contains a separate LICENSE file for this. No need to repeat here.

File: README.md (hunk 1 above)
Comment (samuelsogbesan) [reply README.md]: Thanks for the feedback. I'll go ahead and remove the section until I have the content to add.

File: README.md (hunk 2 above)
Comment (samuelsogbesan) [reply README.md]: Sure thing, I'll remove that section

**Comment by samuelsogbesan** [comment]
@krasimir-google , I've added a new commit with the suggested changes!

**Review by krasimirgg — APPROVED** [review]
LGTM! (Looks good to me!)

**Pull request merged**